frontend/src/components/header/Header.tsx

```diff
@@ -42,9 +42,9 @@ const Header: React.FC<HeaderProps> = ({isLogged}: HeaderProps) => {
 
     const redirectToPage = (isLogged: boolean, navigate: NavigateFunction) => {
         if (isLogged) {
-            navigate(Paths.Home);
-        } else if (location.pathname !== Paths.SignUp) {
-            navigate(Paths.SignIn);
+            navigate(Paths.HOME);
+        } else if (location.pathname !== Paths.SIGN_UP) {
```

Comment:
Later if we want to have forgot password page and maybe anything else, this condition will be complicated and follow your precious rule number 1